### DISCUSSION TITLE
wishing for the possibility to set organization location as a city not only a country (or to be editable)

### DISCUSSION THREAD

**Discussion by eyas-ranjous** [community/community#15762] [Other Feature Feedback, Questions, & Ideas]
<img width="557" alt="org_location" src="https://user-images.githubusercontent.com/6517308/165874134-2006b847-5e69-418b-a01e-f65b7e4c5a19.png">

**Comment by queenofcorgis** [comment]
Thanks for sharing, we appreciate the feedback. I'd love the learn more about the use case behind this request if you wouldn't mind elaborating please :octocat:

**Reply by eyas-ranjous** [reply]
Hi! Well sorry to respond with a question, What's the logic behind that? For example in my profile, location is an editable text, why org location is a country from a list?

**Reply by queenofcorgis** [reply]
I asked because it helps our team better understand our users' needs and can be a factor in determining why our team should work on this 😄 I hope that helps answer your question.

**Reply by eyas-ranjous** [reply]
Well simply I want to add location as a city, not a country.

**Reply by AlexNeises** [reply]
It would also be beneficial to have the ability to add multiple locations. We have teams around the globe and would like to showcase that. Having a text field that we can input whatever makes that possible.

**Reply by jefferyjohn** [reply]
Hi! Our case is that we're a university club, and adding a city helps add much more clarity as to who and where we are. This is particularly useful when requesting partnerships or distinguishing between our different campuses.
